src/components/Form/bookForm.js

```diff
@@ -26,6 +26,7 @@ function Form() {
 
   return (
     <form onSubmit={handleSubmit}>
```

Comment:
- [ ] Kindly make sure to align the Form properly so it aligned with the book container

![Screenshot_20230512_095626](https://github.com/Booze33/book-store/assets/25227022/1dfc58b9-822c-4156-a382-599c176d201b)